src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/contactperson/AbstractContactPersonMapping.kt

```diff
@@ -15,7 +15,9 @@ abstract class AbstractContactPersonMapping(
 
   val whenCreated: LocalDateTime? = null,
 
-) : Persistable<String> {
+) : Persistable<T> {
 
   override fun isNew(): Boolean = new
 }
+
```

Comment:
Convenient default version, all are Strings except one that is a Long for Phones